src/builtins/README.md

```diff
@@ -58,7 +58,7 @@ This looks as follows:
 ]
 ```
 
-Each `observation` actually includes many key-value pairs. The precise content of the `observation` is not known until runtime because it depends on which plugins have been included in the pipeline. Different values have to be treated differently when we upsample in time. The method we use to upsample depends on the `aggregation-method` defined for each key in `units.yml`. 
+Each `observation` actually includes many key-value pairs. The precise content of the `observation` is not known until runtime because it depends on which plugins have been included in the pipeline. Different values have to be treated differently when we upsample in time. The method we use to upsample depends on the `aggregation-method` defined for each key in `units.yml`.
```

Comment:
should it be observation or inputs?

Reply:
it could be `input`. I';ve treated `observation` as the object, rather than using `observation` to refer to individual key/value pairs. wyt?

Reply:
yeah could be. Observation explains better the meaning